content/docs/core/quickstart.md

```diff
@@ -71,7 +71,7 @@ Although identity verification failed, you successfully integrated Pomerium with
 
 :::tip
 
-Because this guide doesn't include a [signing key](/docs/reference/signing-key) in the configuration, identity verification will fail.
+Because this guide doesn't include a [signing key](/docs/reference/signing-key#signing-key) in the configuration, identity verification will fail.
```

Comment:
I think these anchor tag changes can be reverted too, here and in the following files too?

Reply:
I searched for anchors with `#signing-key` and removed it. Good call.